BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -548,6 +591,11 @@ - (void)handleSignInWithCredential:(NSString *)credentialKey value:(id<SBBJSONVa
             keysAndValues[self.reauthTokenKey] = reauthToken;
         }
         
+        if (credentialKey && credentialValue) {
+            keysAndValues[self.credentialKeyKey] = credentialKey;
+            keysAndValues[self.credentialValueKey] = credentialValue;
+        }
```

Comment:
Why is this being added? Can you add a comment? Thanks

Also, looks like this is actually set to the keychain using a different queue from the auth queue? Is there a potential for stuff to get clobbered?

Reply:
Adding a comment.

As for the other thing, until the cleanup code executes at the bottom of this method, this is still protected by the "one-auth-call-to-Bridge-at-a-time" serialization so it won't get called on multiple threads.